spec/operations/story_operations/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe StoryOperations::Create do
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
+  let(:user)        { membership.user }
+  let(:project)     { membership.project }
+  let(:story)       { project.stories.build(story_params) }
+
+  subject { -> { StoryOperations::Create.new.call(story: story, current_user: user) } }
+
+  context 'with valid params' do
+    it { expect { subject.call }.to change { Story.count } }
+    it { expect { subject.call }.to change { Changeset.count } }
+    it { expect(subject.call.value!).to be_eql Story.last }
```

Comment:
also, since you're using `subject`, you can call this `it` block like this: `it { is_expected.to .....` buy I think you need to remove the `proc` definition. You can read more about `subjects` here https://www.betterspecs.org/#subject

Reply:
I tried it now, but it shows a deprecation warning